### DISCUSSION TITLE
Environmental lighting - setting hdr map

### DISCUSSION THREAD

**Discussion by fukalhan** [sceneview/sceneform-android#202] [Q&A]
Hello Thomas,

thank you for adding support for setting custom hdr maps, we are thrilled with this feature. Unfortunately, I'm not able to import any own hdr map I'm trying to use. I have downloaded multiple hdr files from Poly Haven and first tried to add it to resources raw folder, but it doesn't work with it, it throws some numeric exception when trying to read from this location. Then I tried to add the hdr files to public github repo and reference the url in the code, but the hdr file isn't loaded, the model is just black. When I use the two maps from your environmental lights sample, it's working. Can you please help me to find out, how to use my own hdr files? Thank you

**Comment by ThomasGorisse** [comment]
Hi,

I have to duplicate the doc for each file loading functions but here are the infos about the fileLocation format.
https://github.com/SceneView/sceneform-android/blob/ac8efbb0b0764a69732426eba85500ac30389c5c/core/src/main/java/com/gorisse/thomas/sceneform/util/ResourceLoader.kt#L25-L41
Your GitHub file hosting can only be accessed from outside if they are on github.io (from what I remember)

**Reply by fukalhan** [reply]
Thank you very much, the problem was I didn't know how to reference the uri of raw resources, it's all working perfectly now :)
